Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/Settings.java

```diff
@@ -59,6 +61,15 @@
 
   private HashMap<String, IniFile> mIniFiles = new HashMap<>();
 
+  private File mWiimoteProfileFile0;
+  private File mWiimoteProfileFile1;
+  private File mWiimoteProfileFile2;
+  private File mWiimoteProfileFile3;
+  private IniFile mWiimoteProfileIni0 = new IniFile();
+  private IniFile mWiimoteProfileIni1 = new IniFile();
+  private IniFile mWiimoteProfileIni2 = new IniFile();
+  private IniFile mWiimoteProfileIni3 = new IniFile();
```

Comment:
You should use arrays (or some other kind of collection, like a HashMap) for these so you don't need to duplicate code.